Docs: fixed descriptions (#3125)
juicedata/juicefs: docs/en/benchmark/benchmark.md

title: Performance Benchmark
sidebar_position: 1
slug: .

> description: This article describes benchmarking the file system using FIO, mdtest, and the bench command that comes with JuiceFS.

Reviewer: 不用列太详细? 将来还会增加, 比如 `juicefs benchmark`

```suggestion
description: This article describes how to do benchmarking on JuiceFS.
```
